Runtime/BaselibNetworkInterface.cs

```diff
@@ -747,7 +760,7 @@ private static unsafe int EndSendMessage(ref NetworkInterfaceSendHandle handle,
                 messagePtr,
                 1u,
                 &error);
-            if (error.code != ErrorCode.Success)
+            if (error.code != ErrorCode.Success || count != 1u)
```

Comment:
This is part of the fix. The low level Linux UDP networking code wasn't properly signaling an error (didn't end up being our issue but is a legit bug, Simon said).